### DISCUSSION TITLE
Is there a way to customize the AST names autogenerated?

### DISCUSSION THREAD

**Discussion by cwdaniel** [TriPSs/nestjs-query#299] [Q&A] [answered]
Currently when auto-generating names, it generates AST as "CreateOneT", "CreateManyT", "UpdateOneT", "UpdateManyT", where the AST output groups by action type for mutation. Id like to invert this. So that liked domains are grouped. TCreateOne, TCreateMany, TUpdateOne, TUpdateMany, ..etc. Is there functionality for this already? Or do I have to create custom resolvers to handle creating these names?

**Comment by GP4cK** [comment, marked as answer]
You can customise the name of individual resolvers like this: https://tripss.github.io/nestjs-query/docs/graphql/resolvers/#override-endpoint-name
But AFAIK, there's no way to change this globally.

**Reply by cwdaniel** [reply]
Thanks. I missed this completely and I'm not sure how. This is sufficient enough for me.

I think I might have some awkwardness with auto generated relation tables but in most cases I'm creating custom resolvers so it's probably not relevant.
